.github/workflows/raydp.yml

```diff
@@ -63,7 +63,13 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           pip install wheel
-          pip install pytest koalas tensorflow torch tabulate ray[tune] xgboost_ray grpcio-tools
+          if [ "$(uname -s)" == "Linux" ]
+          then
+            pip install torch==1.8.1+cpu -f https://download.pytorch.org/whl/torch_stable.html
```

Comment:
Why we need this?

Reply:
pip install torch in Linux will download ~800M, while torch-cpu is only ~100M. And we don't need gpu anyway.